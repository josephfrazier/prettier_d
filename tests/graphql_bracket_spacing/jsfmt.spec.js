run_spec(__dirname, ["graphql"]);
run_spec(__dirname, ["graphql"], { bracketSpacing: false });
