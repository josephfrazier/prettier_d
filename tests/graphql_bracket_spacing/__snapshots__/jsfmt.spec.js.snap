// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bracket_spacing.graphql 1`] = `
====================================options=====================================
parsers: ["graphql"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
fragment Visit on HighlightedVisit
  @argumentDefinitions(
    count: {type: "Int", defaultValue: 20}
  ) {
  name
}

=====================================output=====================================
fragment Visit on HighlightedVisit
  @argumentDefinitions(count: { type: "Int", defaultValue: 20 }) {
  name
}

================================================================================
`;

exports[`bracket_spacing.graphql 2`] = `
====================================options=====================================
bracketSpacing: false
parsers: ["graphql"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
fragment Visit on HighlightedVisit
  @argumentDefinitions(
    count: {type: "Int", defaultValue: 20}
  ) {
  name
}

=====================================output=====================================
fragment Visit on HighlightedVisit
  @argumentDefinitions(count: {type: "Int", defaultValue: 20}) {
  name
}

================================================================================
`;
